.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /deploy
 /screenshots
+/src/examples-highlights
```

Comment:
Hm, it feels a little weird to have built files in the src folder. Can we move this to the root of the repo?